src/Account/Forms/RegistrationFormType.php

```diff
@@ -0,0 +1,193 @@
+<?php declare(strict_types=1);
+
+namespace VSV\GVQ_API\Account\Forms;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\IntegerType;
+use Symfony\Component\Form\Extension\Core\Type\PasswordType;
+use Symfony\Component\Form\Extension\Core\Type\RepeatedType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Translation\TranslatorInterface;
+use Symfony\Component\Validator\Constraints\Email;
+use Symfony\Component\Validator\Constraints\GreaterThan;
+use Symfony\Component\Validator\Constraints\Length;
+use Symfony\Component\Validator\Constraints\NotBlank;
+use Symfony\Component\Validator\Constraints\Regex;
+
+class RegistrationFormType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        /** @var TranslatorInterface $translator */
+        $translator = $options['translator'];
+
+        $builder
+            ->add(
+                'email',
+                TextType::class,
+                [
+                    'constraints' => [
+                        new Email(
```

Comment:
@stejes misschien kunnen in ons Email value object dezelfde constraint gebruiken? Weet niet of dat makkelijk lukt om een constraint aan te maken en dan een value te valideren...